### PR TITLE
Add google form for the projects

### PR DESCRIPTION
**Pull request by ali4006**

### CODE REVIEW

**Comment by glatard** [comment]
Hi @ali4006 thanks for doing this! However, we had discussed in the shared TODO list that project submission should rather happen through a github issue template, do you think you could update the form accordingly?

**Comment by ali4006** [comment]
> Hi @ali4006 thanks for doing this! However, we had discussed in the shared TODO list that project submission should rather happen through a github issue template, do you think you could update the form accordingly?

I thought the form is more user-friendly. 
Ok, I'll change it to a Github issue template.

**Comment by ali4006** [comment]
Done!

**Review by gkiar — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/mtl-brainhack-global-project-proposal.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+> Please fill in the below table to submit your project proposal!
+
+| Field                | Value            |
+|:--------------|:-------------|
+| Project Name |   *Your project name here*    |
+| Description    |   *Your description here* |
+| URL(s)            |   *Your relevant URL(s) here* |
+| Contributors' Brainhack Slack handle |   *Your slack handle here* |
```
Comment (glatard) [inline .github/ISSUE_TEMPLATE/mtl-brainhack-global-project-proposal.md:17]: Should be Mattermost handle now...

Comment (gkiar) [reply .github/ISSUE_TEMPLATE/mtl-brainhack-global-project-proposal.md:17]: Merged this already, but in my updated PR I switched it to Github handle (seems more useful)

**Pull request merged**